setup.cfg

```diff
@@ -1,3 +1,10 @@
 [flake8]
-exclude = migrations,docs/conf.py,.tox
-max-line-length = 100
+exclude = .tox
+max-line-length = 120
+
+[bdist_wheel]
+universal = 1
+
+[isort]
+profile = black
+multi_line_output = 3
```

Comment:
We might as well exclude ``.tox`` here as well.  Maybe "skip_glob = .tox"?

Reply:
Oh, it looks like that directory is already skipped: https://pycqa.github.io/isort/docs/configuration/options/#skip I had no idea!

Reply:
Excellent.